// ng2-slim-loading-bar/__ivy_ngcc__/index.js
// Copyright (C) 2016 Sergey Akopkokhyants
// This project is licensed under the terms of the MIT license.
// https://github.com/akserg/ng2-slim-loading-bar
import { NgModule } from "@angular/core";
import { SlimLoadingBarComponent } from './src/slim-loading-bar.component';
import { SlimLoadingBarService } from './src/slim-loading-bar.service';
import * as ɵngcc0 from '@angular/core';
export * from './src/slim-loading-bar.component';
export * from './src/slim-loading-bar.service';
export var SlimLoadingBarModule = (function () {
    function SlimLoadingBarModule() {
    }
    SlimLoadingBarModule.forRoot = function () {
        return {
            ngModule: SlimLoadingBarModule,
            providers: [SlimLoadingBarService]
        };
    };
    /** @nocollapse */
    SlimLoadingBarModule.ctorParameters = function () { return []; };
SlimLoadingBarModule.ɵfac = function SlimLoadingBarModule_Factory(t) { return new (t || SlimLoadingBarModule)(); };
SlimLoadingBarModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: SlimLoadingBarModule });
SlimLoadingBarModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ providers: [SlimLoadingBarService] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(SlimLoadingBarModule, { declarations: function () { return [SlimLoadingBarComponent]; }, exports: function () { return [SlimLoadingBarComponent]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(SlimLoadingBarModule, [{
        type: NgModule,
        args: [{
                declarations: [SlimLoadingBarComponent],
                exports: [SlimLoadingBarComponent],
                providers: [SlimLoadingBarService]
            }]
    }], function () { return []; }, null); })();
    return SlimLoadingBarModule;
}());
